collect_app/src/main/java/org/odk/collect/android/fragments/dialogs/CustomDatePickerDialog.java

```diff
@@ -82,6 +82,12 @@ public Dialog onCreateDialog(Bundle savedInstanceState) {
                 .create();
     }
 
+    @Override
+    public void onDestroyView() {
```

Comment:
This works but it's not a proper solution. You already have ViewModel and it is a component you should use to store data and make sure it persists. I mean passing data from a widget to dialog using bundle is ok but then you should unpack it and store in your viewmodel.